actionpack/test/dispatch/request_test.rb

```diff
@@ -1061,7 +1061,7 @@ class RequestParameterFilter < BaseRequestTest
 
       filter_words << 'blah'
       filter_words << lambda { |key, value|
-        value.reverse! if key =~ /bargain/
```

Comment:
Why this change was needed?

Reply:
@prathamesh-sonpatki that's a fair point